### DISCUSSION TITLE
Turning off the menu bar icon

### DISCUSSION THREAD

**Discussion by nmcnulty** [iina/iina#5211] [Q&A] [answered]
I'm unable to locate an option to turn off the IINA menu bar icon under Settings. Can anyone provide me with guidance on how to deactivate it?

Menu bars are easily crowded these days, particularly for those with a Mac that has a 'notch' on the display. I have to figure there's a way to turn the IINA icon off, but I've had no luck finding it.

TIA!

**Comment by low-batt** [comment]
If IINA has a menu bar icon it is news to this developer. I am a junior developer, so it is possible there is a feature I have not encountered. Can you share a screenshot showing the icon? From where did you download the version of IINA you are running?

**Reply by nmcnulty** [reply]
Thanks for the quick reply. The menu bar icon is only visible when the program is loaded. Quitting IINA (usually) causes the icon to disappear, though only after after several seconds. 

I'm confident there are occasions where exiting the program does not eliminate the menu bar icon, though. Yesterday, when I posted my question, I was finding that exiting the program had no effect on the menu bar.

I've attached a screenshot of the icon.
<img width="551" alt="Screenshot 2024-10-14 at 3 51 33 PM" src="https://github.com/user-attachments/assets/e7a74e62-594b-45bc-9eb3-30074bfbe2e4">

**Comment by low-batt** [comment, marked as answer]
Ah. I know what that is. I didn't recognize what you were referring to because I have it shut off, for exactly the reason you stated. I have a MacBook Pro with the camera housing that intrudes into the menu bar and yes, space is tight. This behavior is coming from the macOS Control Center. To disable this follow these instructions:

- Click on  `System Settings…` under the  menu
- The system settings panel appears
- On the left side of the panel click on `Control Center`
- In the `Control Center Modules` section look for the `Now Playing` module
- Change the setting for that module to `Don't Show in Menu Bar`
![control-center](https://github.com/user-attachments/assets/7c44b1fc-5cfd-415f-8dfb-a54cc2aaf763)

The Now Playing module will still be available from the menu bar using the Control Center icon:
![now-playing](https://github.com/user-attachments/assets/3328e4e8-953a-4836-bf40-fb71e47536c8)

Another way to suppress this is to completely turn off IINA's integration with the macOS media keys.
- Start IINA
- Click on  `Settings…` under the `IINA` menu
- The settings panel appears
- On the left side of the panel click on `Key Bindings`
- On the right side of the panel at the top look for the `Use system media control` setting

Unchecking that IINA setting disables IINA's integration with `Now Playing`.

**Reply by nmcnulty** [reply]
Solved! Was completely unaware of that feature in macOS. Thanks so much for taking the time to walk me through how to turn it off.
